### ISSUE TITLE
Missing Validation Set

### ISSUE THREAD

**Issue by Vincentcent1** [KaiyangZhou/deep-person-reid#13]
Is there a validation set used for choosing the best model before testing the accuracy on the test set?
From what I see from the code, the model with Best Rank 1 is chosen based on test set result. Won't this mean that the Best Rank 1 result is overfitting on the test set?

**Comment by KaiyangZhou** [comment]
Thank you for pointing out. We are planning to add the validation.

**Comment by joel-huang** [comment]
i think test set is the name of the validation set
